# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/VAAPIUtils.h
# Commit: 85974aa6ae0b
# Full Hash: 85974aa6ae0bf20be02a09ecfcd3aa6f27ebcb91
# Author: stransky <stransky@redhat.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1751987
# File Overlap Count: 3
# Description:
#   Bug 1751987 [Linux] Implement VAAPIUtils to get basic VA-API info r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139299
# ==============================================================================

diff -r 469777a5cb87 -r 85974aa6ae0b widget/gtk/VAAPIUtils.h
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/widget/gtk/VAAPIUtils.h	Wed Feb 23 11:47:06 2022 +0000
@@ -0,0 +1,11 @@
+/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* vim:set ts=2 sw=2 sts=2 et cindent: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+namespace mozilla::widget {
+
+bool VAAPIIsSupported();
+
+}